### PR TITLE
more on db schema

### PR DESCRIPTION
**Pull request by lucadealfaro**

### CODE REVIEW

File: analysis/db_schema.py [hunk 1]
```diff
@@ -59,6 +61,25 @@ def define_tables(uri, migrate_enabled = False, fake_migrate_all=False):
...
+    db.define_table(
+        'revision_block',
+        Field('page_id', 'reference page'),
+        # These fields are used to know in which order to stitch blocks
+        # together, and also, if there are holes between them.
+        # Id of the last revision before the block
+        Field('prev_revision_id', 'integer'), # long??
+        Field('next_revision_id', 'integer'),
+        # To put blocks in chronological order by page.
+        Field('initial_timestamp', 'datetime'),
```
Comment (ddavisscott) [inline analysis/db_schema.py:73]: what would be better? Type timestamp or datetime? Considering revisions will be international, would UTC  help us?

File: analysis/db_schema.py [hunk 2]
```diff
@@ -59,6 +61,25 @@ def define_tables(uri, migrate_enabled = False, fake_migrate_all=False):
         Field('reputation', 'double'),
     )
 
+    db.define_table(
+        'revision_block',
+        Field('page_id', 'reference page'),
+        # These fields are used to know in which order to stitch blocks
+        # together, and also, if there are holes between them.
+        # Id of the last revision before the block
+        Field('prev_revision_id', 'integer'), # long??
```
Comment (ddavisscott) [inline analysis/db_schema.py:70]: I am not sure, I think we get back an integer?

File: analysis/db_schema.py [hunk 3]
```diff
@@ -36,6 +36,8 @@ def define_tables(uri, migrate_enabled = False, fake_migrate_all=False):
     db.define_table(
         'revision',
         Field('rev_id', 'integer'),
+        # In which block it is.
+        Field('block_id', 'reference revision_block'),
         # user id on the wikipedia for this user.
         Field('user_id', 'integer'),
         Field('rev_date', 'datetime'),
```
Comment (ddavisscott) [inline analysis/db_schema.py:43]: Same here. We need to decide the time format.

**Review by ddavisscott — APPROVED** [review]

**Comment by lucadealfaro** [comment]
I think we should represent all times/dates using the datetime class in the database, and the datatime module in Python, using UTC time.  In this way: 

- The time in the database is in a readable format (easier than second since 1970) and we can make sense of it. 
- The Python timedelta class easily enables the computation of time differences. 

We can then convert to localtime if needed only during visualization.

**Pull request merged**